<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="indLabel">
             <property name="text">
              <string>&amp;L</string>
             </property>
             <property name="buddy">
              <cstring>indValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="indValue"/>
           </item>
           <item>
            <widget class="QComboBox" name="indUnit">
             <item>
              <property name="text">
               <string>uH</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>nH</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mH</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="capLabel">
             <property name="text">
              <string>&amp;C</string>
             </property>
             <property name="buddy">
              <cstring>capValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="capValue"/>
           </item>
           <item>
            <widget class="QComboBox" name="capUnit">
             <item>
              <property name="text">
               <string>nF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>uF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mF</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="freqLabel">
             <property name="text">
              <string>&amp;f</string>
             </property>
             <property name="buddy">
              <cstring>freqValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="freqValue"/>
           </item>
           <item>
            <widget class="QComboBox" name="freqUnit">
             <item>
              <property name="text">
               <string>kHz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MHz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hz</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="indImpLabel">
             <property name="text">
              <string>ZL [ohm]</string>
             </property>
             <property name="buddy">
              <cstring>indImpValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="indImpValue"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="capImpLabel">
             <property name="text">
              <string>ZC [ohm]</string>
             </property>
             <property name="buddy">
              <cstring>capImpValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="capImpValue"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="opLabel">
         <property name="text">
          <string>Calculate for:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="opCombo">
         <item>
          <property name="text">
           <string>f</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="calcButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="quitButton">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElectronic_Calculator">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuElectronic_Calculator"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>indValue</tabstop>
  <tabstop>indUnit</tabstop>
  <tabstop>capValue</tabstop>
  <tabstop>capUnit</tabstop>
  <tabstop>freqValue</tabstop>
  <tabstop>freqUnit</tabstop>
  <tabstop>opCombo</tabstop>
  <tabstop>calcButton</tabstop>
  <tabstop>indImpValue</tabstop>
  <tabstop>capImpValue</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
